CHANGELOG.md

```diff
@@ -605,10 +604,11 @@ Other minor additions
   [1+a%n]≤1+m⇒[a%n]≤m : 0 < suc n % d → suc n % d ≤ suc m → n % d ≤ m
 
   0/n≡0           : 0 / n ≡ 0
-  n/1≡a           : n / 1 ≡ n
+  n/1≡n           : n / 1 ≡ n
```

Comment:
The name of this proof also needs to be changed.
https://github.com/agda/agda-stdlib/blob/ef31b4238066c921f3d4b66feffdf97202aebd02/src/Data/Nat/DivMod.agda#L133-L134

Reply:
Thanks, changed!